src/plugins/dashboard/common/types.ts

```diff
@@ -26,6 +28,21 @@ import {
   RawSavedDashboardPanel730ToLatest,
 } from './bwc/types';
 
+import { GridData } from './embeddable/types';
+export type PanelId = string;
+export type SavedObjectId = string;
+
+export interface DashboardPanelState<
```

Comment:
Had to move this to `common` from `public/type/i_container`
Didn't move the whole file because it contains references to client side types